docs/static/breaking-changes-80.asciidoc

```diff
@@ -0,0 +1,64 @@
+[[breaking-8.0]]
+=== Breaking changes in 8.0
+Here are the breaking changes for 8.0.
+//NOTE: The notable-breaking-changes tagged regions are re-used in the
+//Installation and Upgrade Guide
+
+// tag::notable-breaking-changes[]
+[discrete]
+[[security-on-8.0]]
+===== Secure communication with {es} 
+{ls} must establish a secure connection using an {es} self-signed SSL certificate before it can transfer data to an on-premise {es} cluster. 
+{es} generates its own default self-signed Secure Sockets Layer (SSL) certificates at startup. 
+{ls} must have a copy of the {es} CA and the self-signed certicate to establish a secure connection. 
+
+Our hosted {ess} simplifies safe, secure communication between Logstash and Elasticsearch. 
+{ess} uses standard publicly trusted certificates, and therefore setting a cacert is not necessary.
```

Comment:
Mayyyybe too much of a nitpick?

```suggestion
{ess} uses certificates signed by standard publicly trusted certificate authorities, and therefore setting a cacert is not necessary.
```

Reply:
Not at all! I like it. Thanks for this and the other suggestions.